File: JAVASCRIPT PROGRAMMING/Test62.js
"use strict";
// 62. REVERSE NUMBER
const Reverse = (num) => {
    let rev = 0;
    let NUM = num;
    while (num != 0) {
        let rem = num % 10;
        rev = (rev * 10) + rem;
        num = Math.floor(num / 10);
    }
    console.log(`Reverse Of ${NUM} Is: ${rev}`);
};
Reverse(123);
Reverse(1234);
Reverse(12345);

File: PYTHON PROGRAMMING/Test62.py
# 62. SUM OF DIGITS OF A NUMBER
from math import *
def SumOfDigit(num):
    sum = 0
    NUM = num
    while num != 0:
        rem = num % 10
        sum += rem
        num = floor(num / 10)
    print(f"Sum Of Digits Of {NUM} Is: {sum}")
SumOfDigit(153)
SumOfDigit(123456789)